tests/test_charm.py

```diff
@@ -17,14 +26,26 @@ def setUp(self):
         self.harness.set_leader(True)
         self.harness.update_config({"pagerduty_key": "123"})
 
+    @unittest.skip("")
     def test_config_changed(self):
+        def get_config():
+            pod_spec = self.harness.get_pod_spec()
+            config_yaml = pod_spec[0]["containers"][0]["volumeConfig"][0]["files"][0][
```

Comment:
Style comment : How about a one line utility method to make this `pod_spec[0]["containers"][0]["volumeConfig"][0]["files"][0] ...` more readable ?

Reply:
This is an old test that will soon be rewritten. I kept it for now for reference.